spec/lib/stellar/operation_spec.rb

```diff
@@ -12,6 +12,53 @@
 
 end
 
+describe Stellar::Operation, ".path_payment" do
+  it "works" do
+    destination = Stellar::KeyPair.random
+    amount = [:alphanum4, "USD", Stellar::KeyPair.master, 10]    
+    with = [:alphanum4, "EUR", Stellar::KeyPair.master, 9.2]
+
+    op = Stellar::Operation.path_payment(
+      destination: destination,
+      amount: amount,
+      with: with
+    )
+
+    expect(op.body.arm).to eql(:path_payment_strict_receive_op)
```

Comment:
In this test and the other two similar tests can we test more than the `arm` field?

Reply:
good call! it's a good idea since we could messed up the semantics on the arguments. I just added some test to make sure that the operation represents what we want.